### ISSUE TITLE
Will you release the code of the transfer learning process?

### ISSUE THREAD

**Issue by MarcWong** [ishann/aeroscapes#2]
Really interested in your multi-source process of transfer learning.

**Comment by ishann** [comment]
Hey, apologies for the delay in replying. As the paper indicates, the training strategy is quite simple. We freeze the task-specific layers in pre-trained models and finetune them on the AeroScapes dataset. A linear regressor is trained to combine the softmax predictions from each network. This strategy should allow you to train your own network. Unfortunately, it is not possible for us to release the code/ models.
